.github/workflows/main-workflow.yml

```diff
@@ -12,10 +12,10 @@ jobs:
         uses: actions/setup-dotnet@v1
       - run: |
           cd tshirts
-          bash ../.github/workflows/expectfail.sh dotnet run
+          #bash ../.github/workflows/expectfail.sh dotnet run

```

Comment:
Removing this will miss the running of the tests. Retain `dotnet run` in the workflow. Same for the binaries below as well